### ISSUE TITLE
[Rust] async not highlighted

### ISSUE THREAD

**Issue by dyedgreen** [sublimehq/Packages#2381]
- Sublime Version: 3.2.2
- OS Version: 10.15.4

``` rust
// async/ await is not highlighted
async fn test() {
    other_fn().await;
}
```

**Comment by michaelblyons** [comment]
Perhaps look into #2305?

**Comment by dyedgreen** [comment]
Yeah, I believe that should contain the relevant fix 😃

Should we leave this issue open and close it when #2305 lands? (I was previously searching the open issues and this didn't come up, which is why I opened the new issue, I'd imagine this would help point people in the right direction 😅)

**Comment by deathaxe** [comment]
Fixed by #2305
